UPGRADE-2.8.md

```diff
@@ -213,6 +213,15 @@ Form
        }
    }
    ```
+   
+ * The option "options" of the CollectionType has been renamed to "entry_options".
+   The usage of the option "options" is deprecated as of Symfony 2.8 and will be 
+   removed in Symfony 3.0.
+
+ * The option "type" of the CollectionType has been renamed to "entry_type".
+   The usage of the option "type" is deprecated as of Symfony 2.8 and will be 
+   removed in Symfony 3.0. As a value for the option you need to provide now
```

Comment:
"should" provide (instead of "need") as its not mandatory in 2.8 yet. And grammar: "As a value for the option you need to provide the fully-qualified class name (FQCN) now as well."